shortcodes/epfl_social_feed/instagram_view.php

```diff
@@ -0,0 +1,59 @@
+                  <div
+                      style="padding-top: 8px;">
+                      <div style=" color:#3897f0; font-family:Arial,sans-serif; font-size:14px; font-style:normal; font-weight:550; line-height:18px;"> View this post on Instagram</div>
```

Comment:
Pas de `<?php esc_html_e('View this post on Instagram', 'epfl-shortcodes'); ?>` ?